Integrations/integration-ATD.yml

```diff
@@ -998,28 +1024,28 @@ script:
     - name: fileName
       description: The name of the file
     outputs:
```

Comment:
Make sure CS is aware of this change as it breaks BC

Reply:
talked with gavriel, changing type of output to string seems to be ok